<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsUserWidget</class>
 <widget class="QWidget" name="CwmsUserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WMS Benutzereditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel9">
       <property name="text">
        <string>ID</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pqleUserId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel4">
       <property name="text">
        <string>Vorname</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_pqleName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>Nachname</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_pqleSurname"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel5">
       <property name="text">
        <string>Login</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_pqleLogin"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>E-Mail</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="m_pqleEmail"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="m_pqchbActive">
       <property name="text">
        <string>Aktiv</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pqchbAdmin">
       <property name="text">
        <string>Administrator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pqchbSchemeIndependent">
       <property name="text">
        <string>Schemaunabhängig</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Gruppenmitgliedschaften</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="m_pqlvGroups">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Gruppe</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="m_pqpbAddGroup">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../wmsqml/src/wmsqml.qrc">
             <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbRemoveGroup">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../wmsqml/src/wmsqml.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pqwSchemeAssignment">
      <attribute name="title">
       <string>Schemazugehörigkeit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QListWidget" name="m_pqlwSchemes"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_pqpbAddScheme">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../wmsqml/src/wmsqml.qrc">
             <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbRemoveScheme">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../wmsqml/src/wmsqml.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>201</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>201</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/usermanagerresources.qrc">:/png/WOGRAConsulting-Logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbChangePassword">
       <property name="toolTip">
        <string>Passwort Ã¤ndern</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/usermanagerresources.qrc">
         <normaloff>:/png/change-password.png</normaloff>:/png/change-password.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbUpdate">
       <property name="toolTip">
        <string>Aktualisieren</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/usermanagerresources.qrc">
         <normaloff>:/png/ok.png</normaloff>:/png/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_pqleName</tabstop>
  <tabstop>m_pqleLogin</tabstop>
  <tabstop>m_pqpbChangePassword</tabstop>
  <tabstop>m_pqpbUpdate</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/usermanagerresources.qrc"/>
  <include location="../../wmsqml/src/wmsqml.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbChangePassword</sender>
   <signal>clicked()</signal>
   <receiver>CwmsUserWidget</receiver>
   <slot>ChangePasswordClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbUpdate</sender>
   <signal>clicked()</signal>
   <receiver>CwmsUserWidget</receiver>
   <slot>UpdateCLickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddGroup</sender>
   <signal>clicked()</signal>
   <receiver>CwmsUserWidget</receiver>
   <slot>AddGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveGroup</sender>
   <signal>clicked()</signal>
   <receiver>CwmsUserWidget</receiver>
   <slot>RemoveGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddScheme</sender>
   <signal>clicked()</signal>
   <receiver>CwmsUserWidget</receiver>
   <slot>AddScheme()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveScheme</sender>
   <signal>clicked()</signal>
   <receiver>CwmsUserWidget</receiver>
   <slot>RemoveScheme()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddGroup()</slot>
  <slot>RemoveGroup()</slot>
  <slot>AddScheme()</slot>
  <slot>RemoveScheme()</slot>
 </slots>
</ui>
